qml/CredentialCard.qml

```diff
@@ -29,7 +29,20 @@ Pane {
                  toolBar.searchField.text.toLowerCase()) > -1 ? true : false
 
     background: Rectangle {
-        color: app.isDark() ? app.defaultDarkLighter : app.defaultLightDarker
+        color: itemColor()
+    }
+
+    MouseArea {
+        anchors.fill: parent
+        onClicked: credentialCard.GridView.isCurrentItem ? credentialCard.GridView.view.currentIndex = -1 : credentialCard.GridView.view.currentIndex = index
+    }
+
+    function itemColor() {
+        if (credentialCard.GridView.isCurrentItem) {
+            return app.isDark() ? "#444444" : "#eeeeee"
```

Comment:
Maybe give these colors names as well, similar to the others?

Reply:
agreed, fixing it